catalog/video/video-mpegts-stream/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -77,7 +77,14 @@
             <property name="stanag4609Processor">
                 <bean class="org.codice.alliance.libs.klv.Stanag4609ProcessorImpl">
                     <argument>
-                        <bean class="org.codice.alliance.libs.klv.OffsetCenterPostProcessor"/>
+                        <bean class="org.codice.alliance.libs.klv.ListPostProcessor">
+                            <argument>
+                                <list>
+                                    <bean class="org.codice.alliance.libs.klv.TrimmingPostProcessor"/>
+                                    <bean class="org.codice.alliance.libs.klv.OffsetCenterPostProcessor"/>
```

Comment:
I'll have to look at that carefully to make sure the order of the post processors do not matter. I would also want to keep ListPostProcessor because I want my code to only deal with a single PostProcessor at a high level, not a list of processors. At this time, I don't really expect other modules/bundles to register their own post processors.